src/app/public/modules/i18n/locale-provider.ts

```diff
@@ -12,9 +12,15 @@ import {
 
 @Injectable()
 export class SkyAppLocaleProvider {
+  public get defaultLocale(): string {
+    return SkyAppLocaleProvider._defaultLocale;
+  }
+
+  private static _defaultLocale = 'en-US';
```

Comment:
I had to do the static property this way, since TypeScript doesn't handle inherited static properties that well:
https://stackoverflow.com/questions/34502195/using-static-property-in-inherited-class-method